### PR TITLE
Added jet mode functionality to API

### PR DESCRIPTION
**Pull request by ankushduacodes**
Jet mode functionality seems to be known by different name at different models of AC
On my AC it is known as Himalayas mode

Please Add Ankush Dua to Contributors list

### CODE REVIEW

**Comment by ankushduacodes** [comment]
finally it was accepted.... i was having issues with trailing spaces for last 3 commits....
please merge my pull request

**Comment by ankushduacodes** [comment]
i am very new to GitHub and git.... please don't thing i was spamming commits....
thanks

File: wideq/ac.py [hunk 1]
```diff
@@ -7,6 +7,28 @@
 from .core import FailedRequestError
 
 
+class ACJetMode(enum.Enum):
+    """ To turn on the jet mode. This mode
```
Comment (sampsyo) [inline wideq/ac.py:11]:
```suggestion
    """To turn on the jet mode. This mode
```

File: wideq/ac.py [hunk 2]
```diff
@@ -7,6 +7,28 @@
...
+    3. Fan speed is set to HIGH
+    (@AC_MAIN_WIND_STRENGTH_HIGH_W)
+    after jet mode turns off
+    """
+
+    OFF = "@OFF"
+    COOL_JET = "@COOL_JET"
+    HEAT_JET = "@HEAT_JET"
+    DRY_JET = "@DRY_JET_W"
+    HIM_COOL = "@HIMALAYAS_COOL"
```
Comment (sampsyo) [inline wideq/ac.py:29]: Maybe some simpler names on the Python side would suffice here? Like `OFF`, `COOL`, `HEAT`, `DRY`, and `HIMALAYAS`.

File: wideq/ac.py [hunk 3]
```diff
@@ -7,6 +7,28 @@
...
+    dry or heat mode(for a certain amount of time)
+    depending on what you choose
+
+    This mode Overrides following setting:
+    1. Vertical swing is set to @100
+    2. Temperature gets set to 18 after
+    jet mode turns off
+    3. Fan speed is set to HIGH
+    (@AC_MAIN_WIND_STRENGTH_HIGH_W)
+    after jet mode turns off
```
Comment (sampsyo) [inline wideq/ac.py:22]: This docstring is wrapped rather narrowly—it can be 72 characters wide to be easier to read.

File: wideq/ac.py [hunk 4]
```diff
@@ -235,6 +257,13 @@ def get_zones(self):
 
         return self._get_config('DuctZone')
 
+    def set_jet_mode(self, jet_opt):
+        """Set jet mode to value form `ACJetMode` enum
```
Comment (sampsyo) [inline wideq/ac.py:261]:
```suggestion
        """Set jet mode to a value from the `ACJetMode` enum.
```

**Comment by digginsa** [comment]
Any idea why Wideq does not return any results for my AC units?

File: wideq/ac.py (hunk 2 above)
Comment (ankushduacodes) [reply wideq/ac.py]: done

File: wideq/ac.py (hunk 3 above)
Comment (ankushduacodes) [reply wideq/ac.py:18]: done

**Comment by sampsyo** [comment]
Wonderful; thank you for adding this!! :smiley: :tada: :rocket:

**Pull request merged**